web/src/fonts.css

```diff
@@ -1,13 +1,25 @@
 /* Font variables */
-
 :root {
   /* Font families */
-  --font-primary: "Arial", sans-serif;
-  --font-secondary: "Georgia", serif;
+  --font-primary: "Cambria", sans-serif;
+  --font-secondary: "Cochin", serif;
```

Comment:
i can see there being merge conflicts when this happens, it will be good to communicate with @Priscilla-MacGatus and align the values for these css variables